UoaWDCC/AEYB
# Aeyb/#132 assign roles to users

**Closes:** #132 

**What was done:**
- Redesign of roles page
- Assign roles to users
- Viewing user roles

## **How to test:**
- Add and delete roles from users
- Ensure all 'tabs' work

**Screenshots of UI changes (optional):**
![image](https://github.com/UoaWDCC/aeyb-monitor/assets/108048091/457eb81f-b688-4f79-b0a7-9c8425a4ca12)
![image](https://github.com/UoaWDCC/aeyb-monitor/assets/108048091/8a6ff7d7-96fe-4b62-be49-efd61185365f)
![image](https://github.com/UoaWDCC/aeyb-monitor/assets/108048091/707bb51e-2b0f-4e12-8c6f-9286df4be16f)
![image](https://github.com/UoaWDCC/aeyb-monitor/assets/108048091/4e40907a-72d7-40e1-b297-564cde485c8e)
